fay.lua

```diff
@@ -62,54 +62,45 @@ function Sertify.new(testInfo)
 		
 		self.toBeEmpty = function(self)
 			local evaluation
-			if type(self.initialValue) == "string" then
-				evaluation = self.initialValue:len() == 0
-			elseif type(self.initialValue) == "table" then
+			if type(self.initialValue) == "string" or type(self.initialValue) == "table" then
 				evaluation = #self.initialValue == 0
 			else
-				evaluation = false
+				evaluation = true
```

Comment:
The test shouldn't pass here. initialValue can be a number.
This should be false.